_posts/2017-08-28-upgrade-to-rails-3-1.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title:  "Upgrade Rails from 3.0 to 3.1"
+date: 2017-08-28 11:02:00
+categories: ["rails", "upgrades"]
+author: "luciano"
+---
+
+This is the second article of our [Upgrade Rails series](https://www.ombulabs.com/blog/tags/upgrades). We will be covering the most important aspects that you need to know to update your Ruby on Rails application from version 3.0 to 3.1. If you are in an older version, you can take a look at our [previous article](https://www.ombulabs.com/blog/rails/upgrades/upgrade-to-rails-3-rails-upgrades.html).
```

Comment:
@lubc Please add a link to `Ruby on Rails` (http://rubyonrails.org) and to `version 3.0` (http://guides.rubyonrails.org/3_0_release_notes.html)